OrderService/src/main/java/org/mybatis/jpetstore/controller/OrderController.java

```diff
@@ -62,37 +65,43 @@ else if (cart != null) {
 
     @PostMapping("/newOrder")
     public String newOrder(Order order, @RequestParam(required = false) boolean shippingAddressRequired, @RequestParam(required = false) boolean confirmed, @RequestParam(required = false) boolean changeShipInfo, @RequestParam String csrf, HttpServletRequest req, HttpSession session) {
+
         if (csrf == null || !csrf.equals(session.getAttribute("csrf_token"))) {
             String msg = "This is not a valid request";
             req.setAttribute("msg", msg);
             return "common/Error";
         }
+
         Order sessionOrder = (Order) session.getAttribute("order");
+
         if (shippingAddressRequired) {
             changeBillInfo(sessionOrder, order);
             session.setAttribute("order", sessionOrder);
             return "order/ShippingForm";
-        } else if(!confirmed) {
+        }
+
+        else if(!confirmed) {
             if (changeShipInfo)
                 changeShipInfo(sessionOrder, order);
             session.setAttribute("order", sessionOrder);
             return "order/ConfirmOrder";
-        } else if (order != null) {
+        }
+
+        else if (order != null) {
```

Comment:
```suggestion
else if (sessionOrder != null)
```
어짜피 갱신된 주문 정보는 sessionOrder에 있기 때문에 sessionOrder가 null인지 체크하면 될 것 같아요.
그리고 아직 수정하지 않으신 것 같긴 하던데 insertOrder(sessionOrder)만 넘기도록 하면, insertOrder() 메소드 내부에서 if (order != null) 체크를 할 필요가 없을 것 같아요.